src/components/SharedComponents/ObservationViews/Toolbar.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import type { Node } from "react";
+import React from "react";
+import { Pressable, View } from "react-native";
+import Icon from "react-native-vector-icons/MaterialCommunityIcons";
+
+import { viewStyles } from "../../../styles/observations/obsList";
+
+type Props = {
+  isExplore: boolean,
+  isLoggedIn: ?boolean,
+  syncObservations: Function,
+  setView: Function
+}
+
+const Toolbar = ( {
+  isExplore,
+  isLoggedIn,
+  syncObservations,
+  setView
+}: Props ): Node => (
+  <>
+    {!isExplore && (
+      <View style={viewStyles.toggleButtons}>
+        {isLoggedIn && (
+        <Pressable onPress={syncObservations}>
+          <Icon name="sync" size={30} />
+        </Pressable>
+        )}
+      </View>
+    )}
```

Comment:
All of this `isExplore` stuff is is probably a situation where you want to invert control: the parent component could pass in some extra buttons as props and Toolbar wouldn't have to worry about things like whether someone is logged in.

Reply:
I imagine we'll overhaul this code when we try to get the Explore screen working again. Right now, Explore seems broken enough that I'm not sure how the toolbar looks at all.